app/Platform/Models/Game.php

```diff
@@ -161,6 +161,24 @@ public function shouldBeSearchable(): bool
         return false;
     }
 
+    // == temp subset detection
+
+    public function getParentGameId(string $title, int $consoleId, int $gameId): ?int
```

Comment:
```
GET http://localhost:64000/API/API_GetGameExtended.php?z=...&y=...&i=24511
```

`ParentGameID` is null on this branch, but is 24453 in `master`.

Reply:
The design of the function is worth thinking about some more:

* `getParentGameId()` should not require any arguments - it's already attached to the game that its given arguments are coming from.
* I think the `Game::where()` call can be entirely eliminated, which should improve performance because this query uses a `LIKE`, which skips database indexes.

Reply:
New title check code by @wescopeland implemented, legacy db lookup removed from getParentGameIdFromGameTitle() in favor of result passed from new function in Game model